javascript-crash-course.md

```diff
@@ -5,6 +5,34 @@ permalink: /javascript-crash-course/
 isStaticPost: true
 ---
 <img class="img-responsive feature-image" src="{{ site.baseurl }}/img/posts/js-bg.jpg" style="display:none">
+### Advanced
+
+Welcome to Women Techmakers Berlin's Advanced JavaScript winter course!
+
+The course will spread over 5 lectures during March through May, where we will dive deeper in JS, following our last year's Beginners course. If you missed it - you can catch up, as all of our talks are available below.
+You are also welcome to join the JS Slack (#js-crash-course) in the [Women Techmakers Berlin Slack](https://join.slack.com/t/womentechmakersberlin/shared_invite/enQtMjQ1ODcwNTIyMTAxLTY1NmVjMGQzOTgxNmI0ZWUwYWNlNDg0MjkyYjEwNDY5ZjVlMzU1ZDZlMjc1MGUzNzRhZWM3YzYxMDVmNWYzNjQ), and ask your questions - to catch up on last year's material, or to ask question from the latest lecture.
+This year we will keep uploading lectures, so don't worry if you have to miss one.
+
+In addition to the lectures, we will meet to hack on our projects on a bi-weekly basis, just you, your project, the instructors and a whole-lotta-mate!
+
+
+##### Meeting 1: Design Patterns
+An introduction to popular software design patterns and how they can be utilized with JavaScript. In this session, we talk about what a design pattern is, what anti-patterns are, how they emerge and when to make use of one. We talk about some of the Gang of Four patterns and their applications.
+
+##### Meeting 2: Authentication and middleware
```

Comment:
Let's keep the lecture titles capitalized, e.g. `Middleware` here, and some more changes in the following titles